middleware/basicauth/basicauth_test.go

```diff
@@ -39,10 +41,15 @@ func Test_Middleware_BasicAuth(t *testing.T) {
 		},
 	}))
 
-	//nolint:forcetypeassert,errcheck // TODO: Do not force-type assert
 	app.Get("/testauth", func(c *fiber.Ctx) error {
-		username := c.Locals("username").(string)
-		password := c.Locals("password").(string)
+		username, ok := c.Locals("username").(string)
+		if !ok {
+			t.Fatal("missing username")
```

Comment:
better to use assertequal

Reply:
Should be part of another MR.

Reply:
> Should be part of another MR.

It can be done in this PR since it was added here

Reply:
Sorry for the delay. How would you use assertequal here? We want to retrieve the user + pass from locals.

Reply:
> Sorry for the delay. How would you use assertequal here? We want to retrieve the user + pass from locals.

`utils.AssertEqual(t, true, ok)`